src/cli/one_helper/onevm_helper.rb

```diff
@@ -713,9 +634,9 @@ def ssh(args, options)
             options[:ssh_opts].nil? ? opts = '' : opts = options[:ssh_opts]
 
             if opts.empty?
-                exec(*%W[ssh #{login}@#{ip} -p #{port} #{cmd}])
+                exec('ssh', "#{login}@#{ip}", '-p', port.to_s, cmd.to_s)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

A potential code injection vulnerability has been detected where untrusted input is passed to a method that may execute arbitrary code. This allows attackers to inject and execute malicious code within the application, potentially leading to unauthorized access or data breaches. To mitigate this risk, properly sanitize and validate all user input before processing. Avoid passing untrusted input to methods that can execute code. Consider using safer alternatives like parameterized queries or controlled methods for handling user input. Learn More - https://cwe.mitre.org/data/definitions/94.html.